src/Misc/layoutroot/run.sh

```diff
@@ -15,7 +15,7 @@ while :;
 do
     "$DIR"/run-helper.sh $*
     returnCode=$?
-    if [[ $returnCode == 1 ]]; then
```

Comment:
Should we just change the exit code here? https://github.com/actions/runner/pull/1708/files#diff-869ba9f7fc8ba69126cf55bb734923c0a217d1b731213e48243fffb297b45b30R7

Reply:
Choose something like `3` for non-retryable exit codes and use that convention as needed so we can create an a denylist for what errors are not retryable

Reply:
We spoke in person